src/engine/LocalVocab.h

```diff
@@ -100,6 +102,29 @@ class LocalVocab {
       std::ranges::copy(vocab.otherWordSets_, inserter);
       *inserter = vocab.primaryWordSet_;
     }
+
+    // Also merge the `vocabs` `LocalBlankNodeManager`s, if they exist.
+    using LocalBlankNodeManager =
+        ad_utility::BlankNodeManager::LocalBlankNodeManager;
+    using sharedLbnm = std::shared_ptr<const LocalBlankNodeManager>;
+
+    auto lbnmView =
+        vocabs |
+        std::views::transform([](const LocalVocab& vocab) -> const sharedLbnm {
+          return vocab.localBlankNodeManager_;
+        });
```

Comment:
@joka921 You mentioned that this should obviously return a const&, however this results in "Returning reference to local temporary object" warning and segfaults. I've tried to understand why it doesn't work in this specific case (as it usually should work), but i can't find a solution.

Reply:
Thee reason is the type mismatch between `shared_ptr<LBNM>` and `shared_ptr<const LBNM>` (which are convertible, but not the same type).
So you could write `-> decltype(auto)` or `-> const shared_ptr<LBNM>` to avoid the copy AND the issues with the temporary.